source/extensions/filters/http/cache/http_cache.h

```diff
@@ -252,9 +252,13 @@ class HttpCache {
   //
   // This is called when an expired cache entry is successfully validated, to
   // update the cache entry.
+  //
+  // The on_complete callback should be called with true if the update is successful,
```

Comment:
s/should be/is/

Reply:
I've made this change, but I'm not entirely comfortable with it - this is an interface against which the person reading the documentation is likely implementing a cache instance (since there's no other reason to be looking at this particular code). "Is" sounds like something that already happens. I would favor "must be", as a description of what the implementer needs to make happen. But if you want "is", you can have "is". :)